.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -7,7 +7,6 @@ about: 'Report a general framework issue. Please ensure your Laravel version is
 - Laravel Version: #.#.#
 - PHP Version: #.#.#
 - Database Driver & Version:
-- Are you using `PDO::ATTR_EMULATE_PREPARES => true`: y/n
```

Comment:
Why has this been removed?

Reply:
This was removed in 7a4c6bc

Reply:
🤔 Not quite sure why that's showing up in this PR, let me know if this is an issue